### ISSUE TITLE
question only

### ISSUE THREAD

**Issue by spirogg** [MacWarrior/clipbucket-5.0#35]
Sorry but can you give steps to upgrade again? 

- last time I update all files - but not install folder : 
-  I had an issue that said : it was in maintenance mode and could not figure out how to get it to go away ?
 - So I just reinstalled everything over again ? 

**but would be nice not to reinstall and learn proper way to upgrade :)**

**Comment by MacWarrior** [comment]
Hi
To update : 
- Update files (Just the way you did)
- Remove /upload/files/temp/install.me (If you copied it)
- Apply SQL manually (Check in /upload/cb_install/sql/upgrade_5.X.sql

CB isn't designed for continuous integration ; I've personnaly automated files updates, but I always apply SQL manually (Most of the time, there is no SQL to apply when updating)

I'll push a script to make easiest the files update

**Comment by spirogg** [comment]
That would be great if you can push us trip to update would be awesome even though it's pretty straightforward the way you showed me thank you so much

**Comment by spirogg** [comment]
Do I just select the SQL file and import it to my database ? I'm using cpanel

**Comment by spirogg** [comment]
I can give details to server too if you like to test on different server than yours. Centos 7 with  cpanel it's not an issue for give you details of server I trust you :)

**Comment by spirogg** [comment]
Ok I think I have it right. I changed the file .sql to correct tbl_prefix 
Then went to phpmyadmin and uploaded .sql file and imported. Then I went to admin area and select check box and click save. Now it shows selected. So I think it is correct.
- Because I try to select check mark and save before import .sql file and it would not stay saved with check mark. 

So I'm guessing it is good.  

Thank you.
